test/e2e/common/node/pod_resize.go

```diff
@@ -949,7 +949,7 @@ func doPodResizeErrorTests(f *framework.Framework) {
 //       Above tests are performed by doSheduletTests() and doPodResizeResourceQuotaTests()
 //       in test/e2e/node/pod_resize.go
 
-var _ = SIGDescribe("Pod InPlace Resize Container", framework.WithSerial(), feature.InPlacePodVerticalScaling, "[NodeAlphaFeature:InPlacePodVerticalScaling]", func() {
+var _ = SIGDescribe("Pod InPlace Resize Container", framework.WithSerial(), feature.InPlacePodVerticalScaling, func() {
```

Comment:
if is beta end enabled by default and does not depend on any external factor, you may remove `feature.InPlacePodVerticalScaling` so it gets exercised with all the common jobs ... because this feature will be exercised in all common clusters in prod

Reply:
Thanks. Done.